currencies/src/lib.rs

```diff
@@ -153,6 +156,11 @@ decl_module! {
             <CurrencyIdentifiers>::mutate(|v| v.push(cid));
             <Locations>::insert(&cid, &loc);
             <Bootstrappers<T>>::insert(&cid, &bootstrappers);
+
```

Comment:
this should be kept here, because ultimatlely, the number of newbie tickets may be a custom currency parameter

Reply:
hmm, actuelly, it doesn't matter. may be more intuitive to have this in ceremonies as well....

Reply:
Sorry, I am confused. What do you refer to with `this`? If it is the bootstrappers, I would also vote to put them in the `Ceremonies` pallet. What does the `Currencies` pallet need them for?

Furthermore, we never iterate through them, only perform lookups.  This implies the `BootstrapperNewbieTickets` double_map can actually replace the `Bootrappers` map, thereby gaining a slight performance improvement, see [Maps as sets](https://substrate.dev/recipes/map-set.html).